Apps/W1/External File Storage - Azure Blob Service Connector/app/src/ExtBlobStoConnectorImpl.Codeunit.al

```diff
@@ -0,0 +1,508 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.ExternalFileStorage;
+
+using System.Text;
+using System.Utilities;
+using System.Azure.Storage;
+using System.DataAdministration;
+
+codeunit 4560 "Ext. Blob Sto. Connector Impl." implements "External File Storage Connector"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+    Permissions = tabledata "Ext. Blob Storage Account" = rimd;
+
+    var
+        ConnectorDescriptionTxt: Label 'Use Azure Blob Storage to store and retrieve files.';
+        NotRegisteredAccountErr: Label 'We could not find the account. Typically, this is because the account has been deleted.';
+        MarkerFileNameTok: Label 'BusinessCentral.FileSystem.txt', Locked = true;
+
+    /// <summary>
+    /// Gets a List of Files stored on the provided account.
+    /// </summary>
+    /// <param name="AccountId">The file account ID which is used to get the file.</param>
+    /// <param name="Path">The file path to list.</param>
+    /// <param name="FilePaginationData">Defines the pagination data.</param>
+    /// <param name="Files">A list with all files stored in the path.</param>
```

Comment:
```suggestion
    /// <param name="TempFileAccountContent">A list with all files stored in the path.</param>
```

Reply:
Same issue 2 other places - but don't worry, I've fixed those in my internal PR.